app/v3/controllers/SubmitForeignPropertyBsasController.scala

```diff
@@ -113,25 +120,37 @@ class SubmitForeignPropertyBsasController @Inject()(val authService: EnrolmentsA
 
   private def errorResult(errorWrapper: ErrorWrapper) =
     errorWrapper.error match {
-      case BadRequestError |
-           NinoFormatError |
-           CalculationIdFormatError |
-           CustomMtdError(ValueFormatError.code) |
-           RuleTypeOfBusinessIncorrectError |
-           CustomMtdError(RuleIncorrectOrEmptyBodyError.code) |
-           CustomMtdError(RuleCountryCodeError.code) |
-           CustomMtdError(CountryCodeFormatError.code) |
-           CustomMtdError(RuleDuplicateCountryCodeError.code) |
-           RuleBothPropertiesSuppliedError |
-           CustomMtdError(RuleBothExpensesError.code) => BadRequest(Json.toJson(errorWrapper))
-      case RuleSummaryStatusInvalid |
-           RuleSummaryStatusSuperseded |
-           RuleAlreadyAdjusted |
-           RuleResultingValueNotPermitted |
-           RuleOverConsolidatedExpensesThreshold |
-           RulePropertyIncomeAllowanceClaimed => Forbidden(Json.toJson(errorWrapper))
+      case _
+          if errorWrapper.containsAnyOf(
+            BadRequestError,
+            NinoFormatError,
+            CalculationIdFormatError,
+            TaxYearFormatError,
+            RuleTaxYearRangeInvalidError,
```

Comment:
It makes sense to have  `RuleTaxYearRangeInvalidError` here, but it is not in the business spec, so should it be here?

Reply:
It should be in the spec as it goes hand-in-hand with TaxYearFormatError. (I will fix spec.)